janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerProvider.java

```diff
@@ -60,4 +63,28 @@ public List<EdgeLabel> getEdgeLabels() {
             .stream(management.getRelationTypes(org.janusgraph.core.EdgeLabel.class).spliterator(), false)
             .map(GrpcUtils::createEdgeLabelProto).collect(Collectors.toList());
     }
+
+    public VertexCompositeGraphIndex getVertexCompositeGraphIndexByName(String indexName) {
+        JanusGraphIndex graphIndex = management.getGraphIndex(indexName);
+        if (graphIndex == null) {
+            return null;
+        }
+        if (!Vertex.class.isAssignableFrom(graphIndex.getIndexedElement())) {
+            return null;
```

Comment:
Same here, I think an error would be better as this method shouldn't be called for an edge index. (Should also be tested.)